lib/Sema/CSStep.h

```diff
@@ -519,7 +519,7 @@ template <typename P> class BindingStep : public SolverStep {
       if (CS.isDebugMode()) {
         auto &log = getDebugLogger();
         log << "(attempting ";
-        choice->print(log, &CS.getASTContext().SourceMgr);
+        choice->print(log, &CS.getASTContext().SourceMgr, CS.solverState->depth * 2 + 2);
```

Comment:
An idea for a future refactoring - `depth * 2` pattern appears in a lot of places in the solver, instead it could be packaged into `getCurrentIdent()` method on `SolverState`, that way if we were to change the computation it would have to be done in just one place.